### DISCUSSION TITLE
wallbash broke itself

### DISCUSSION THREAD

**Discussion by Madarastion** [prasanthrangan/hyprdots#1920] [Q&A] [answered]
![resim](https://github.com/user-attachments/assets/ba50e375-bb0e-484f-a260-9c98693a6576)

I just selected light mode on wallbash selector and now everything is broken. I tried Hyde restore but couldnt solve it. How can i fix this?

**Comment by kRHYME7** [comment, marked as answer]
themeswitch.sh n

**Reply by Madarastion** [reply]
oh thank you that solved the issue instantly
